proto/bucket/where_is.proto

```diff
@@ -0,0 +1,72 @@
+syntax = "proto3";
+package sonrio.sonr.bucket;
+
+option go_package = "github.com/sonr-io/sonr/x/bucket/types";
+
+message WhereIs {
+  // DID of the created bucket.
+  string did = 1;
+
+  // Creator of the new bucket
+  string creator = 2;
+
+  // Label of the new bucket.
+  string label = 3;
+
+  // Visibility of the new bucket.
+  BucketVisibility visibility = 4;
+
+  // Role of the creator of the new bucket.
+  BucketRole role = 5;
+
+  // IsActive flag of the new bucket.
+  bool is_active = 6;
+
+  // JWKs of the new bucket.
+  map<string, bytes> jwks = 7;
```

Comment:
This should be a 2D map. I'd suggest a message for the Jwk map:
```
message AclJwks {
  map<string, bytes> encrypted_jwks = 1;
}

map<string, AclJwks> content_acl = 1;

Reply:
agree with @mcjcloud comment